examples/node/.gitignore

```diff
@@ -1 +0,0 @@
-*.js
```

Comment:
why?

Reply:
please revert

Reply:
This is following Yury's suggestion to merge all .gitignores into a single .gitignore for all examples.